core-web/apps/dotcms-ui/src/app/portlets/shared/directives/dot-dropdown.directive.ts

```diff
@@ -34,20 +34,22 @@ export class DotDropdownDirective {
     }
 
     /**
-     *Array of option to translate LABEL_NAME and assign to Dropdown
+     *Array of options to translate LABEL_NAME and assign to Dropdown
      *
      * @param {Array<ExperimentsSelectOption<string>>} options - Options of Dropdown
      * @memberof DotDropdownDirective
      */
     @Input()
     set dotOptions(options: Array<ExperimentsSelectOption<string>>) {
```

Comment:
`ExperimentsSelectOption` this kills the reusability of this directive, you either make it reusable or move it to experiments.